### PR TITLE
add more information to README

### PR DESCRIPTION
**Pull request by gedl**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,6 +4,70 @@ dice-fairlink is a JDBC driver designed to connect to the read replicas of an AW
 The driver will periodically obtain a description of the cluster and dispatch connections to each read replica
 on round-robin fashion.
 
+# Why do we need dice-fairlink (TL/DR version)?
+Because in many cases Aurora will not evenly distribute the connections amongst all the available read replicas.
+[image]
+
+# Why do we need dice-fairlink ?
+Using AWS Aurora clusters with database connection pools is an possible use case. A possible configuration is to point
```
Comment (stykiaz) [inline README.md]: a possible use

File: README.md [hunk 2]
```diff
@@ -4,6 +4,70 @@ dice-fairlink is a JDBC driver designed to connect to the read replicas of an AW
 The driver will periodically obtain a description of the cluster and dispatch connections to each read replica
 on round-robin fashion.
 
+# Why do we need dice-fairlink (TL/DR version)?
+Because in many cases Aurora will not evenly distribute the connections amongst all the available read replicas.
```
Comment (stykiaz) [inline README.md]: we probably should say why the links are not distributed properly by aurora.

File: README.md [hunk 3]
```diff
@@ -4,6 +4,70 @@ dice-fairlink is a JDBC driver designed to connect to the read replicas of an AW
...
+[image]
+
+# Why do we need dice-fairlink ?
+Using AWS Aurora clusters with database connection pools is an possible use case. A possible configuration is to point
+a connection pool to the cluster's read endpoint. AWS claims ([here](https://aws.amazon.com/blogs/aws/new-reader-endpoint-for-amazon-aurora-load-balancing-higher-availability/),
```
Comment (stykiaz) [inline README.md]: read-only

File: README.md (hunk 2 above)
Comment (gedl) [reply README.md]: It's explained elsewhere (not on the TL/DR section)

**Pull request merged**